spec/lib/metasploit/cache/referencable/ephemeral/referencable_references_spec.rb

```diff
@@ -217,8 +217,8 @@
                 build_added
               end
 
-              it 'is a new record' do
-                expect(built_reference).to be_new_record
+              it 'is a newly created record' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.